### PR TITLE
Improve Metadock's Dedaub Support

### PR DESCRIPTION
**Pull request by ju61n**
This PR adds support for/improves:

- Dedaub support for Arbitrum;
- Dedaub's transaction debugger;
- Update Dedaub's decompile button link;
- And, adds viewing storage on Dedaub's Contract Library.

### CODE REVIEW

File: src/content/etherscan/components/DedaubStorageShortcut/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  const pathname = DEDAUB_SUPPORT_DIRECT_LIST.find(
+    item => item.chain === chain
+  )?.pathname
+
+  return (
+    <>
+      {pathname && (
+        <li
+          className="nav-item snap-align-start"
+          style={{ cursor: 'pointer' }}
```
Comment (npetridis) [inline src/content/etherscan/components/DedaubStorageShortcut/index.tsx:23]: Perhaps we should add this CSS property in a class to preserve consistency

File: src/content/scans/feat-scripts/dedaub-storage.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const genDedaubStorageShortcut = async (chain: string) => {
+  const mainAddress = pickAddress(window.location.pathname)
+  if (!mainAddress) return
+
+  const isContract = !!document.querySelector(
+    '#ContentPlaceHolder1_li_contracts'
+  )
+
+  if (!isContract) {
+    console.log('IS not Contract')
```
Comment (npetridis) [inline src/content/scans/feat-scripts/dedaub-storage.tsx:17]: Do we need this log?

**Review by npetridis — APPROVED** [review]
Please take a look at the minor comments. Looks good to me 👍🏼

File: src/content/scans/feat-scripts/dedaub-storage.tsx (hunk 2 above)
Comment (cong1223) [reply src/content/scans/feat-scripts/dedaub-storage.tsx:17]: No, we dont't

File: src/content/etherscan/components/DedaubStorageShortcut/index.tsx (hunk 1 above)
Comment (cong1223) [reply src/content/etherscan/components/DedaubStorageShortcut/index.tsx:23]: Thank you for your suggestion, I will review and confirm this

Comment (ju61n) [reply src/content/etherscan/components/DedaubStorageShortcut/index.tsx]: Currently, within the code base there is `common.less` at `src/common/styles/common.less` and it contains the following CSS class:

https://github.com/blocksecteam/metadock/blob/722e478d530713b1a0d19dfeb5bec4c8af1fc54e/src/common/styles/common.less#L47-L49

This class is globally available without `import`.

**Comment by cong1223** [comment]
Hi,

Thank you for submitting your Pull Request. To ensure the cleanliness and consistency of our code repository, we kindly request you to resubmit your changes to the develop branch.

If you encounter any issues during this process, feel free to reach out to us.

Thank you for your cooperation!

Best regards

**Pull request closed without merging**